app/reducers/views/channel.js

```diff
@@ -236,6 +252,7 @@ export default combineReducers({
     displayName,
     drafts,
     loading,
+    refreshing,
```

Comment:
think of moving this to a reducer that can be easily blacklisted and perhaps we should do the same with `loading`